### PR TITLE
update latest dispatcher and dependencies

### PR DESCRIPTION
**Pull request by adictovirtual**
update latest dispatcher and dependencies

### CODE REVIEW

**Comment by adictovirtual** [comment]
@chrisabrams any thoughts on this ?

File: package.json [hunk 1]
```diff
@@ -4,63 +4,65 @@
...
   "dependencies": {
-    "babel-runtime": "^6.20.0",
-    "express": "^4.14.0",
-    "paradigm-slugs": "0.0.4",
-    "paradigm-taxonomies": "0.0.10",
-    "structure-dispatcher": "0.0.32",
-    "structure-driver": "0.0.3",
-    "structure-root-controller": "0.0.3",
-    "structure-root-model": "0.0.19"
+	"babel-runtime": "^6.20.0",
```
Comment (chrisabrams) [inline package.json]: What's up with the spacing here? It was fine before.

File: package.json [hunk 2]
```diff
@@ -4,63 +4,65 @@
...
+	"supertest": "^2.0.0"
   },
   "scripts": {
-    "compile": "node_modules/.bin/babel src -d dist -D -s",
-    "prepublish": "npm run compile",
-    "start-reqlite": "node_modules/forever/bin/forever start node_modules/reqlite/lib/node.js --port-offset 1 -s",
-    "stop-reqlite": "node_modules/forever/bin/forever stop node_modules/reqlite/lib/node.js --port-offset 1 -s",
-    "test": "export NODE_ENV=test && gulp stop-reqlite && npm run start-reqlite && gulp t && gulp stop-reqlite",
-    "test:debug": "export LOG_LEVEL=debug && npm test"
+	"compile": "node_modules/.bin/babel src -d dist -D -s",
```
Comment (chrisabrams) [inline package.json]: Same for here. Undo this spacing change.

File: test/integration/routes.js [hunk 3]
```diff
@@ -33,6 +56,8 @@ describe('Routes', function() {
 
     var res = await new MockHTTPServer()
       .post(`/api/${process.env.API_VERSION}/tags`)
+      .set('organizationid',orgId)
```
Comment (chrisabrams) [inline test/integration/routes.js]: These also need a space.

**Comment by chrisabrams** [comment]
@adictovirtual also, package.json is missing some dependencies.

**Pull request merged**